lib/shopify-cli/tasks/tunnel.rb

```diff
@@ -22,30 +22,17 @@ def call(ctx)
 
       def stop(ctx)
         @ctx = ctx
-        if running?
```

Comment:
Should we keep this `if running?` (or change how `stop` works) otherwise we might be telling them it can't be stopped when it wasn't running in the first place

Reply:
This will tell them that it was successfully stopped in all circumstances but yes I can put it back